server/controller/src/main/resources/template/image-build.yaml

```diff
@@ -27,7 +27,7 @@ spec:
               swcli runtime copy cloud://server/project/$SW_PROJECT/runtime/$SW_RUNTIME_VERSION .;
               swcli runtime extract --target-dir /tmp/extract/ $SW_RUNTIME_VERSION;
               cp -r /tmp/extract/** /opt/starwhale.user/runtime;
-              swcli runtime dockerize -t gxx-test --push --dry-run --use-starwhale-builder --reset-qemu-static $SW_RUNTIME_VERSION;
+              swcli runtime dockerize -t sw-build --push --dry-run --use-starwhale-builder --reset-qemu-static $SW_RUNTIME_VERSION;
```

Comment:
`-t sw-build --push --use-starwhale-builder --reset-qemu-static` seems like useless?

Reply:
fixed, but `-t sw-build` is necessary, others can remove.